lib/open-sesame.coffee

```diff
@@ -5,8 +5,12 @@ LEADING_SLASH_REGEX = /^\//
 OPEN_COMMAND        = 'open-sesame:open-file-under-cursor'
 
 getPathUnderCursor = ->
-  atom.workspace.getActiveTextEditor()?.getWordUnderCursor
-    wordRegex: PATH_REGEX
+  editor = atom.workspace.getActivePaneItem()
+  text = editor?.getSelectedText()
+  unless text.length > 0
```

Comment:
I tried something like that earlier and couldn't get it to work but your code here works great.